cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -321,358 +325,252 @@ describe('Credentials', () => {
-          cy.verifyPageTitle('Create Credential');
-          cy.getByDataCy('name').should('be.visible').type(credentialName);
-          cy.wait('@getCredentialTypes').then(() => {
-            cy.singleSelectByDataCy('credential_type', `${item.name}`, true);
-            if (Array.isArray(item.required)) {
-              (item.required as { field: string; dataCy: string }[]).forEach((credentialType) => {
-                switch (item.name) {
-                  case 'HashiCorp Vault Secret Lookup':
-                    cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
-                    cy.selectDropdownOptionByResourceName('api-version', 'v1');
-                    break;
-                  case 'Ansible Galaxy/Automation Hub API Token':
-                    cy.singleSelectByDataCy('organization', awxOrganization.name);
-                    cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
-                    break;
-                  case 'GPG Public Key':
-                    cy.get('#gpg-public-key').type(`${credentialType.field}`);
-                    break;
-                  case 'Terraform backend configuration':
-                    cy.get('[data-cy="configuration-form-group"]').type(`${credentialType.field}`);
-                    break;
-                  default:
-                    cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
-                }
-              });
-            }
-            cy.intercept('POST', awxAPI`/credentials/`).as('created');
-            cy.getByDataCy('Submit').click();
-            cy.wait('@created');
-            cy.verifyPageTitle(credentialName);
-            cy.clickPageAction('delete-credential');
-            cy.get('#confirm').click();
-            cy.clickButton(/^Delete credential/);
-            cy.verifyPageTitle('Credentials');
-          });
-        });
-      });
-    });
-
-    describe('Credentials: Credential Types Tests', () => {
-      it('can create credential using custom credential type', () => {
-        cy.createAwxCredentialType().then((credType) => {
-          const credentialName = 'E2E Credential ' + randomString(4);
-          cy.navigateTo('awx', 'credentials');
-          cy.clickButton(/^Create credential$/);
-          cy.getByDataCy('name').type(credentialName);
-          cy.singleSelectBy('[data-cy="credential_type"]', credType.name, true);
-          cy.clickButton(/^Create credential$/);
-          cy.verifyPageTitle(credentialName);
-          cy.getByDataCy('name').contains(credentialName);
-          //delete created credential
-          cy.clickPageAction('delete-credential');
-          cy.get('#confirm').click();
-          cy.clickButton(/^Delete credential/);
-          cy.verifyPageTitle('Credentials');
-          cy.deleteAwxCredentialType(credType);
-        });
-      });
-
-      it('cannot edit vault id field for Vault credential type', () => {
```

Comment:
Was this converted to a component test? If not, why was it removed?

Reply:
It was not removed, title was edited.